Packs/Active_Directory_Query/Integrations/Active_Directory_Query/README.md

```diff
@@ -410,6 +410,9 @@ Enables a previously disabled Active Directory account.
 
 `ad-enable-account`
 
+##### Required Permissions
+Requires `Read userAccountControl` and `write userAccountControl` permissions.
```

Comment:
1. Isn't the problem with the `disable-account` command?
2. What commands require these permissions? Are we sure it's only one command?

Reply:
`disable-account` and `enable-account` commands.